pkg/apis/core/v1beta1/helper/errors.go

```diff
@@ -56,84 +55,6 @@ func (e *ErrorWithCodes) Error() string {
 	return e.err.Error()
 }
 
-var (
```

Comment:
Why did you drop all this? The extensions have not yet been adapted. We should still apply them for the tasks in the shoot flow which interact with the extension resources for backwards-compatibility